benchmark_runner/jupyterlab/templates/summary_report/summary_report_operations.py

```
@@ -188,6 +193,38 @@ def update_bootstorm_data(self, subset: pd.DataFrame, unique_uuid: str):
+    def update_fio_data(self, subset: pd.DataFrame, unique_uuid: str):
+        average_latency_df = \
+            subset.groupby(['uuid', 'ocp_version', 'cnv_version', 'odf_version', 'block_size', 'io_operation'])[
+                'lat_avg_usec'].mean().reset_index()
+        average_iops_df = \
+            subset.groupby(['uuid', 'ocp_version', 'cnv_version', 'odf_version', 'block_size', 'io_operation'])[
+                'total_iops'].mean().reset_index()
+
+        geo_mean_latency = self.__geometric_mean(average_latency_df['lat_avg_usec'])
+        geo_mean_iops = self.__geometric_mean(average_iops_df['total_iops'])
+
+        logger.info('geo_mean_latency', geo_mean_latency)
```

Ruff (0.15.15)

[error] Too many arguments for `logging` format string

(PLE1205)